enterprise/api/v1/urls.py

```diff
@@ -0,0 +1,20 @@
+"""
+URl definitions for enterprise api version 1 endpoint.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from rest_framework.routers import DefaultRouter
+
+from enterprise.api.v1 import views
+
+router = DefaultRouter()  # pylint: disable=invalid-name
```

Comment:
pylint considers module level variables `constants` and expects them to be uppercased as specified [here](https://www.python.org/dev/peps/pep-0008/#constants).